assets/test/specs/form.spec.js

```diff
@@ -220,5 +229,41 @@ describe('Form Validation', function () {
       });
     });
 
+    describe('When I toggle open the grouped input elements, do not select an item in the group and submit', function () {
+      beforeEach(function () {
+        $groupToggle.click();
+
+        $submitButton.click();
+      });
+
+      it('The error should be visible', function () {
+        expect($groupToggle.closest('.form-field-group')).toHaveClass('form-field-group--error');
+      });
+
+
+      it('The form should have the grouped radio required error message', function () {
+        var $errorMessages = $errorSummaryMessages.find('> li');
+        expect($errorMessages.text().indexOf($radioYesGroupElement.attr('data-msg-required'))).not.toBe(-1);
```

Comment:
This feels a little simpler? and is being used elsewhere in the file?
`expect($errorMessages.text()).toBe($radioYesGroupElement.attr('data-msg-required'));`